core/templates/pages/admin-page/activities-tab/admin-prod-mode-activities-tab.component.ts

```diff
@@ -16,20 +16,17 @@
  * @fileoverview Directive for the activities tab in the admin panel when Oppia
  * is in production mode.
  */
+import { Component, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
 
-require('domain/utilities/url-interpolation.service.ts');
-
-angular.module('oppia').directive('adminProdModeActivitiesTab', [
-  'UrlInterpolationService', function(UrlInterpolationService) {
-    return {
-      restrict: 'E',
-      scope: {},
-      bindToController: {},
-      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
-        '/pages/admin-page/activities-tab/' +
-        'admin-prod-mode-activities-tab.directive.html'),
-      controllerAs: '$ctrl',
-      controller: [function() {}]
-    };
-  }
-]);
+@Component({
+  selector: 'admin-prod-mode-activities-tab',
```

Comment:
`oppia-admin-prod-mode-activities-tab` (In angular directives we are going to have oppia prefix)

Reply:
Done!